salt/modules/cmdmod.py

```diff
@@ -245,6 +245,10 @@ def _run(cmd,
     '''
     Do the DRY thing and only call subprocess.Popen() once
     '''
+    # Remove whitespace from strings
+    if isinstance(cmd, six.string_types):
+        cmd = cmd.strip()
+
```

Comment:
@terminalmage 

So, here's the YAML:
```
Failing-Example:
  cmd.run:
    - name: >
        Clear-Host;
        Write-Host "hello salt"
    - shell: powershell
    - runas: testuser01
    - password: PassWord1!
```

I don't know if it's a YAML problem or what, but this is the command after shellex before this fix:
```
Powershell -NonInteractive "Clear-Host; Write-Host \"hello salt\"
"
```
And here's the command after this fix:
```
Powershell -NonInteractive "Clear-Host; Write-Host \"hello salt\""
```

With `runas` on Windows it's not using `subprocess.Popen()` as that doesn't support runas on Windows.